src/main/java/com/google/devtools/build/lib/remote/TracingMetadataUtils.java

```diff
@@ -88,6 +88,21 @@ public static Metadata headersFromCurrentContext() {
     return headers;
   }
 
+  /**
+   * Extracts a {@link RequestMetadata} from a {@link Metadata} if it exists.
+   *
+   * @param headers
+   * @throws {@link IllegalStateException} if a RequestMetadata could not be found in the given
+   *     headers.
+   */
+  public static RequestMetadata extractRequestMetadata(Metadata headers) {
```

Comment:
nit: rename extractRequestMetadata to fromHeaders to be consistent with other methods in the file

Reply:
done.